### ISSUE TITLE
No Panel Indicator on Elementary OS Freya-0.3.2(Ubuntu 14.04) Ayatana

### ISSUE THREAD

**Issue by DroWnThePoor** [nylas/N1#2820]
Simply doesn't appear in my eOS Pantheon wing-panel which uses Ubuntu's Ayatana API.
This is paramount to usefulness of any e-mail client on GNU/Linux. 
Even better would be to have it appear under the "messages/mail" indicator.

##### What operating system are you using?
Elementary OS Freya-0.3.2(Ubuntu 14.04)
##### What version of N1 are you using?

0.4.51

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?

N/A

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?

NO

##### Is the issue reproducible with a particular attachment, message, signature, etc?

Installed via cloning main repository. May affect Ubuntu 14.04 /w Unity as well.

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?

Yes Geary/Evolution/Thunderbird
Please make it to where Nylas N1's indicator appears under the messages/e-mail indicator like so:

![messagesemail](https://cloud.githubusercontent.com/assets/16339020/18351496/636d5268-75a8-11e6-8656-dacc8efab49b.png)

**Comment by DroWnThePoor** [comment]
I have fixed my own issue, and created this in case anyone else should experience this problem.
In order to fix this I simply found the app icon launcher file located in:
/usr/share/applications/nylas.desktop

And changed Exec= /usr/share/nylas/nylas %U
to
Exec=env XDG_CURRENT_DESKTOP=Unity /usr/share/nylas/nylas %U
Restarted Nylas and all was well, hope this helps someone.
Searching Google I couldn't find anyone experiencing this issue on my version (the only ones I saw were from 0.4.5). Cheers.
